.github/workflows/android.yaml

```diff
@@ -54,7 +54,51 @@ jobs:
                   path: |
                    .environment/gn_out/.ninja_log
                    .environment/pigweed-venv/*.log
-            - name: Build android examples
+            - name: Build arm libs
+              timeout-minutes: 2
               run: |
-                ./scripts/run_in_build_env.sh \
-                  "./scripts/build/build_examples.py --platform android build"
+                  ./scripts/examples/android_app.sh
```

Comment:
something is off here: this looks like a revert. Please do not revert changes (I seem to see this in dockerfile updates - I do not expect things like infineon to be reverted by a pybind11 PR)

Reply:
Yup, I screwed something up.  Will probably have to wait until Josh is back from vacation to re-do the PR from scratch.